tools/wurf_options_context.py

```diff
@@ -7,6 +7,9 @@
 
     import waflib.extras.wurf_options
 
+This step can be done automatically by building a custom waf binary and
+using the --prelude statement (see more here http://bit.ly/1Te4mRz).
+
```

Comment:
Unfortunately, the prelude magic fails with the following error:
```
[localhost] local: python waf-light --make-waf --tools=compat15,C:\steinwurf\waf\tools\wurf_common_tools.py,C:\steinwurf\waf\tools\wurf_configure_outp
ut.py,C:\steinwurf\waf\tools\wurf_dependency_bundle.py,C:\steinwurf\waf\tools\wurf_dependency_resolve.py,C:\steinwurf\waf\tools\wurf_git.py,C:\steinwu
rf\waf\tools\wurf_options.py,C:\steinwurf\waf\tools\wurf_resolve_context.py,C:\steinwurf\waf\tools\wurf_standalone.py,C:\steinwurf\waf\python-semver\s
emver.py --prelude='    import waflib.extras.wurf_options'
------> Executing code from the top-level wscript <-----
No function import defined in C:\steinwurf\waf\waf\wscript
```

For the time being, I reverted the corresponding change in fabric. We can take a look at it later.